soda-core/src/soda_core/contracts/api/verify_api.py

```diff
@@ -30,16 +32,53 @@ def verify_contracts_locally(
     publish: bool = False,
     check_paths: Optional[list[str]] = None,
     dwh_data_source_file_path: Optional[str] = None,
+) -> ContractVerificationSessionResult:
+    if not isinstance(contract_file_paths, list):
```

Comment:
NIT: I would check for `str` instead of `list` to make it easier to read:

```suggestion
    if isinstance(contract_file_paths, str):
```